ring/src/lib.rs

```diff
@@ -35,21 +38,36 @@ pub fn find_complement_point<Curve: SWCurveConfig>() -> Affine<Curve> {
     }
 }
 
+pub fn find_random_point<F: PrimeField, P: AffineRepr<BaseField = F>>() -> P {
+    let mut x: u8 = 0;
```

Comment:
This is used only by tests. Please move within the test module below.

Reply:
done in bd362e2